src/services/projects.js

```diff
@@ -502,15 +582,32 @@ module.exports = class ProjectsHelper {
 								error: utils.errorObject(common.BODY, common.CHILDREN),
 							})
 						}
-					})
+					}
 				}
-			})
+			}
 
 			await resourceQueries.updateOne({ id: projectData.id }, { status: common.RESOURCE_STATUS_SUBMITTED })
 			//TODO: For review flow this has to be changed we might need to add further conditions
 			// and Validate those reviewer as well
-			if (bodyData.hasOwnProperty('reviewer_ids') && bodyData.reviewer_ids.length > 0) {
-				let reviewsData = bodyData.reviewer_ids.map((reviewer_id) => ({
+			if (bodyData.reviewer_ids && bodyData.reviewer_ids.length > 0) {
+				const orgReviewers = await userRequests.list(common.REVIEWER, '', '', '', userDetails.organization_id, {
+					user_ids: bodyData.reviewer_ids,
+				})
+				let orgReviewerIds = []
+				if (orgReviewers.success) {
+					orgReviewerIds = orgReviewers.data.result.data.map((item) => item.id)
+				}
+				if (orgReviewerIds.length <= 0) {
+					throw responses.failureResponse({
+						message: 'REVIEWER_IDS_NOT_FOUND',
```

Comment:
@adithyadinesh0412 change error message to No reviewers found. Contact your organization admin will come up

Reply:
https://katha.shikshalokam.org/story-view-1217.html